pcp/PCPMetric.h

```diff
@@ -0,0 +1,179 @@
+#ifndef HEADER_PCPMetric
+#define HEADER_PCPMetric
+/*
+htop - pcp/PCPMetric.h
+(C) 2020-2021 htop dev team
+(C) 2020-2021 Red Hat, Inc.  All Rights Reserved.
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include <ctype.h>
+#include <pcp/pmapi.h>
+#include <stdbool.h>
+#include <sys/time.h>
```

Comment:
Header order, cf. `STYLEGUIDE`

```suggestion
#include <ctype.h>
#include <stdbool.h>
#include <pcp/pmapi.h>
#include <sys/time.h>
```